.gitignore

```diff
@@ -46,3 +46,6 @@ active_record_cheat_sheet.md
 
 # DeepCode
 .dccache
+
+node_modules/
+yarn.lock
```

Comment:
why have you did this? without the lock file, applications are not in sync anymore across systems. i agree with the node modules. but the lock file sets versions, otherwise it can happen, that newer packages will be used, which can have breaking changes.

Reply:
If that is the case, then why is the `yarn.lock` file not in the repo?

Reply:
ok we talked and this was added at some install step. we do not use node packages in this project. so either way of handling is fine for now. let's keep it here.